docker/dev/Dockerfile

```diff
@@ -1,12 +1,74 @@
-FROM lychee-base:latest AS base
+# Later when we use lychee-base, We can just remove this big block of Dockerfile.
```

Comment:
I'm not sure why we are copying the whole docker file. because woth the make command it will build both lychee base and lychee dev. and tag them accordingly